content/kots-cli/download.md

```diff
@@ -5,4 +5,24 @@ title: kots download
 weight: 90120
 ---
 
-Once an application is running and has the Admin Console (kotsadm) deployed with it, you can run `kots download` to retrieve a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation.
+Retrieves a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation. Requires a running KOTS application with the Admin Console (kotsadm).
+
+### Usage
+```bash
+kubectl kots download [app-slug] [flags]
+```
+* _Replace `[app-slug]` with the "slug" of your KOTS application_ 
+* _Provide `[flags]` according to the table below_ 
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--dest` | string |        the directory to store the application in (default "/Users/austin") |
+| `-h, --help`   |  |           help for download |
+| `--kubeconfig` | string |  the kubeconfig to use (default "/Users/austin/.kube/config") |
```

Comment:
The default kubecfg flag is dynamic.  If there is a $KUBECONFIG envvar we use that, else we use $HOMEDIR/.kube/config, except on windows where we use $USERPROFILE/.kube/config.

This is the same logic that kubectl uses to determine the kubeconfig, so maybe we can just refer how that detects kubeconfig?

Reply:
What is displayed is what is rendered by "--help". I'll go ahead and add the helpful details about the flag.

Reply:
Saying default is $KUBECONFIG. If unset, then $HOME/.kube/config. Windows users generally know that $HOME = $USERPROFILE (saying this messed with the column presentation)